### PR TITLE
fix(DB/Creature): Added skills and movement to Razorbeak Gryphon and Razorbeak Skylord

### PR DESCRIPTION
**Pull request by Asthalor**
I added the skills Rend Flesh and Wing Flap on both of them as the source indicates

Closes AzerothCore issue#6798

## Changes Proposed:
-  Added the skill Rend Flesh to use it every 14 seconds
-  Added the skill Wing Flap to uise it every 10 seconds

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6798
- Closes https://github.com/chromiecraft/chromiecraft/issues/1120
- Closes https://github.com/chromiecraft/chromiecraft/issues/1121

## SOURCE:

https://wowpedia.fandom.com/wiki/Razorbeak_Skylord
https://www.wowhead.com/npc=2659/razorbeak-skylord#abilities

## Tests Performed:

- .go c id 2658 and  .go c id 2659
- attack and see if they use them

Small video proof

https://user-images.githubusercontent.com/87535580/126146894-11eef505-8db7-4041-8286-061ab18e8fe0.mp4

## How to Test the Changes:

1.  .go c id 2658 and  .go c id 2659
2. Attack and see if they use them

## Known Issues and TODO List:

- I choosed 14 and 10 seconds. The bleed one is a bleed that last 12 seconds, so i wanted to give some time to use it again. The knockup is quicker so it will not happen both attacks at the same time.
- I dont know much about difficulty, so if the timers need to be changed please advise

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by LukasVolgger — APPROVED** [review]
Ready to test and pls give feedback about the known issues during tests.

**Comment by syssneck** [comment]
Hi, here are my test results and what I found out:
- Razorbeak Gryphon (.go c id 2658) and Razorbeak Skylord (.go c id 2659) are both not moving (they have no random movement). It would be really nice, if you could add this in this PR @Asthalor :)
- Both are now friendly to Alliance characters, but hostile to Horde characters (please fix):
unknown_2021 07 19-19 43_1
- Both are using Rend Flesh and Wing Flap. 
- Rend Flesh does, dependent on the creature level, between 55-65 dmg per tick. This seems reasonable for this level range (not too easy, not too hard)
- Wing Flap works and knocks my characters back a bit (when hit)
- I checked the db, there are indeed only these two Razorbeaks
Test combat video:

https://user-images.githubusercontent.com/68868567/126207142-9054b769-f828-46a0-ad02-4b4f3cc7ada5.mp4

**Comment by LukasVolgger** [comment]
Thx @syssneck for your great test results!
Imo it sounds to be a better behaviour and even more blizzlike then before the PR and this could be merged.

Any objections?

**Comment by syssneck** [comment]
All good, no objections. I will just create another issue for the movement then. Ty!

**Comment by LukasVolgger** [comment]
> All good, no objections. I will just create another issue for the movement then. Ty!

@Asthalor maybe you wanna have a look at this too? It could also be done in this PR by fixing two issues

**Comment by Asthalor** [comment]
> > All good, no objections. I will just create another issue for the movement then. Ty!
> 
> @Asthalor maybe you wanna have a look at this too? It could also be done in this PR by fixing two issues

Hello! Thanks a lot for the testing and feedback @syssneck .
Ive yet to do something related to movements of creatures, i started here 2 or 3 days ago, but i can definively take a look at that if you dont mind waiting a bit more to merge @LukasVolgger

**Comment by syssneck** [comment]
@Asthalor ty for taking a look at it! I think it has something to do with MovementType.

**Comment by Asthalor** [comment]
@syssneck  thanks for the tip! In the end it was not hard :P

Added movement patterns to both of them.
TEST

Razorbeak Gryphon
https://user-images.githubusercontent.com/87535580/126309302-157e51ce-d380-496f-9ec7-de1942191e21.mp4

Razorbeak Skylord

https://streamable.com/cwpljt

File: data/sql/updates/pending_db_world/rev_1626690253021473300.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(93550, 2658, 0, 0, 0, 1, 1, 935, 0, 138.441, -3310.63, 115.738, 1.23146, 350, 5, 0, 2059, 0, 1, 0, 0, 0, '', 0);
+
+-- Added movement for Razorbeak Skylord
+DELETE FROM `creature` WHERE (`id` = 2659) AND (`guid` IN (93612, 93615));
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(93612, 2659, 0, 0, 0, 1, 1, 1148, 0, 133.356, -4032.63, 127.874, 3.52646, 350, 5, 0, 2399, 0, 1, 0, 0, 0, '', 0),
+(93615, 2659, 0, 0, 0, 1, 1, 1148, 0, 200.395, -3662.88, 133.817, 3.43305, 350, 10, 0, 2399, 0, 1, 0, 0, 0, '', 0);
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1626690253021473300.sql:28]: Are you updating the movement type and radius? If so ut would be better to change them with an `UPDATE`

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

**Comment by Asthalor** [comment]
Changed the SQL query and tested again. 
All good on my machine.

**Comment by syssneck** [comment]
Checked in-game, they both wander now. Good job!

**Comment by LukasVolgger** [comment]
Good job @Asthalor and @syssneck !

**Pull request merged**